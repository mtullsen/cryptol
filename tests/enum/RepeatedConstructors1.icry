:load RepeatedConstructors1.cry
