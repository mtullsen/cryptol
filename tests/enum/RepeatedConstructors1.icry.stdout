Loading module Cryptol

Parse error at RepeatedConstructors1.cry:1:10--1:11
  Multiple declarations for `X`
  Other declaration: RepeatedConstructors1.cry:1:14--1:15
